alibi/explainers/cfproto.py

```diff
@@ -701,7 +710,7 @@ def fit(self, train_data: np.ndarray, trustscore_kwargs: dict = None, d_type: st
         else:
             preds = np.argmax(self.predict(train_data), axis=1)
 
-        self.cat_vars_ord = None
+        self.cat_vars_ord = dict()  # type: dict
```

Comment:
Avoids checking for `None` later on when passing to functions that expect a `dict`.